goal_src/jak1/pc/progress-pc.gc

```diff
@@ -1660,18 +1660,10 @@
               (set! (-> *game-info* mode) 'play)
               ;; Start a new game differently if speedrunning mode is active
               (cond ((= (-> *pc-settings* speedrunner-mode?) #t)
```

Comment:
For the same reason as above, I was under the impression that I had to explicitly check because OpenGOAL doesn't really have a true boolean -- am i mistaken and i can just do `(-> *pc-settings* speedrunner-mode?)`?